codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibraryBuildCUDAKernels/.vscode/launch.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do we need this empty file?

Reply:
The file has been replaced with correct content. Done.